config.go

```diff
@@ -102,16 +101,6 @@ func WithInternalServiceDefaults() TLSOption {
 	}
 }
 
-// WithPivotalDefaults is the same as WithInternalServiceDefaults and is only
-// provided for backwards compatibility. These configuration options are now
-// used beyond just Pivotal.
-//
-// Deprecated: Use WithInternalServiceDefaults() instead.
-func WithPivotalDefaults() TLSOption {
```

Comment:
Removing the `WithPivotalDefaults()` looks good to me as there [don't appear to be any usages](http://cs.cf-app.com/search?q=WithPivotalDefaults+-f%3Aparaphernalia+-r%3Aparaphernalia+-r%3Atlsconfig&num=50). As this is a new import path anyway then this can't be a breaking change.